registry.json

```diff
@@ -5419,5 +5419,19 @@
    "telegram": "Dinoaus73",
    "operatorName": "John Dean",
    "github": ""
+  },
+{
+   "ticker": "CAN1",
+   "poolId": "566ebd3406088123f8dcc99630f122520db5d479e661c72b43571627",
+   "social": {
+     "twitter": "canadastakes1",
+     "website": "https://www.canadastakes.ca",
+     "facebook": "",  
+     "instagram": "canadastakes",
+     "youtube": ""
+   },
+   "telegram": "",
```

Comment:
You must have a telegram handle to join and it must be of an individual and not a group or channel